product_variants_no_automatic_creation/views/product_view.xml

```diff
@@ -17,18 +17,54 @@
             <field name="model">product.template</field>
             <field name="inherit_id" ref="product.product_template_only_form_view" />
             <field name="arch" type="xml">
-                <button string="Variant Prices" position="attributes">
+            <xpath expr="//button[@name='103']" position="attributes">
```

Comment:
Its not a number, its a string.. and the name of the button.
According the view (product.template.common.form) 103 is its name
So instead of looking up the text string "Variant Prices" its looking for
the name of the button

[image: Inline afbeelding 1]

and works just fine.
Its hiding the original variant prices button and adds the new one
[image: Inline afbeelding 2]


2015-12-12 7:36 GMT-05:00 Pedro M. Baeza <notifications@github.com>:

> In product_variants_no_automatic_creation/views/product_view.xml
> <https://github.com/odoomrp/odoomrp-wip/pull/1098#discussion_r47431254>:
>
> > @@ -17,18 +17,54 @@
> >              <field name="model">product.template</field>
> >              <field name="inherit_id" ref="product.product_template_only_form_view" />
> >              <field name="arch" type="xml">
> > -                <button string="Variant Prices" position="attributes">
> > +            <xpath expr="//button[@name='103']" position="attributes">
>
> This number won't work in any database. You have to use the same
> expression as in original.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/odoomrp/odoomrp-wip/pull/1098/files#r47431254>.
>